rs-matter-embassy/Cargo.toml

```diff
@@ -60,10 +60,12 @@ sequential-storage = "3"
 embassy-net = { version = "0.6", optional = true, features = ["proto-ipv4", "proto-ipv6", "multicast", "dhcpv4"] }
 edge-nal-embassy = { version = "0.5", optional = true }
 
-trouble-host = { version = "0.1", git = "https://github.com/embassy-rs/trouble" }
+#trouble-host = { version = "0.1", git = "https://github.com/embassy-rs/trouble" }
+trouble-host = { git = "https://github.com/embassy-rs/trouble", branch = "main" }
 bt-hci = "0.2"
 
-openthread = { version = "0.1", git = "https://github.com/ivmarkov/esp-openthread", optional = true, features = ["udp", "srp", "edge-nal"] }
+# openthread = { version = "0.1", git = "https://github.com/ivmarkov/esp-openthread", optional = true, features = ["udp", "srp", "edge-nal"] }
+openthread = { git = "https://github.com/ivmarkov/esp-openthread", branch = "main", optional = true, features = ["udp", "srp", "edge-nal"] }
```

Comment:
Ditto here. `main` is the default branch, so the two deps are in fact the same thing.
Also and in the meantime, we are tracking the `main` branch from the `esp-rs` repo where my changes were upstreamed yesterday.
We are (for now) additionally patching `esp-openthread` to my own "next" branch (which in 99% of the cases should be the same but allows quick bug-fixes without re-patching). We do the same for `rs-matter` btw.